File: api/omp4m/codegen/lib/ompSetNested/ompSetNested.c
#include "ompSetNested.h"
#include "omp.h"

void ompSetNested(int nested)
{
  omp_set_nested(nested);
}

void ompSetNested_initialize(void)
{
}

void ompSetNested_terminate(void)
{
}

File: api/omp4m/ompSetNested.m
function ompSetNested(nested) %#codegen
%Enables or disables nested parallelism.
%
%    ompSetNested(nested)
%
% See also ompGetNested

%#codegen -args {int32(0)}

if ~strcmp(coder.target,'rtw'); return; end

coder.inline('always');
coder.allowpcode('plain')

coder.cinclude('omp.h');

coder.ceval('omp_set_nested', nested);
